File: CoreJava/college-app/src/com/techlab/person/Student.java
package com.techlab.person;

import java.util.Date;

public class Student extends Person {
	private Branch branch;

	public Student(int id, String name, String address, Date dateOfBirth, Branch branch) {
		super(id, name, address, dateOfBirth);
		this.branch = branch;
	}

	public Branch getBranch() {
		return branch;
	}
	
	
}

File: C#Folder/CollegeApp/CollegeApp/Model/Student.cs
using System;

namespace CollegeApp.Model
{
    class Student : Person
    {
        private Branch _branch;

        public Student(int id, string address, DateTime dateOfBirth, Branch branch) : base(id,address,dateOfBirth)
        {
            _branch = branch;
        }

        public Branch Branch { get => _branch; set => _branch = value; }
    }
}
